goentri.com.clickmax.json

```diff
@@ -0,0 +1,22 @@
+{
+  "providerId": "goentri.com",
+  "providerName": "Entri",
+  "serviceId": "clickmax",
+  "serviceName": "ClickMax",
+  "version": 1,
+  "logoUrl": "https://cdn.goentri.com/logo.svg",
+  "description": "Allows user to easily set up dkim record using Entri",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com,app.goentri.com",
+   "variableDescription": "cnamehost is the host for the CNAME and cnamePointsTo is the URL to where the subdomain is pointed ",
+  "sharedProviderName": true,
+  "hostRequired": false,
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "%cnamehost%",
```

Comment:
It would be better for the template to use `@` as host + hostRequired=true and `host` parameter to set the right subdomain